pubsubplus/templates/solaceConfigMap.yaml

```diff
@@ -337,33 +337,11 @@ data:
         echo "`date` INFO: ${APP}-Redundancy state is not yet up."
         rm -f ${FINAL_ACTIVITY_LOGGED_TRACKING_FILE}; exit 1
       fi

```

Comment:
Changes in pubsubplus/templates/solaceConfigMap.yaml should be removed from this PR